### DISCUSSION TITLE
Responsive design

### DISCUSSION THREAD

**Discussion by ImamJanjua** [nativewind/nativewind#900] [General]
Hi, previously i used react-native-responsive-screen for responsive design. so basically i takes the screen dimensions and based on that u can define styles in % so wp(50) ➔ width 50%. How can i now make my design responsive. Do i have to use container queries and or can i still use such libs which i would prefer for the simplicity?

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
